packages/react-core/src/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -136,69 +163,43 @@ import { MenuToggle } from '@patternfly/react-core';
 </React.Fragment>
 ```
 
-### Split button (checkbox)
-    
-```ts file='MenuToggleSplitButtonCheckbox.tsx'
-```
-
-### Split button (checkbox with toggle text)
-    
-```ts file='MenuToggleSplitButtonCheckboxWithText.tsx'
-```
+### Split button toggle with checkbox
 
-### Split button (checkbox, primary)
-    
-```ts file='MenuToggleSplitButtonCheckboxPrimary.tsx'
-```
+To add an action button or other control to a menu toggle, use a split button. A `<MenuToggle>` can be rendered as a split button by adding a `splitButtonOptions` object. Elements to be displayed before the toggle button must be included in the `items` property of `splitButtonOptions`.
 
-### Split button (checkbox, secondary)
-    
-```ts file='MenuToggleSplitButtonCheckboxSecondary.tsx'
-```
+The following example shows a split button with a <MenuToggleCheckbox>.
 
-### Split button (action)
-    
-```ts file='MenuToggleSplitButtonAction.tsx'
+```ts file='MenuToggleSplitButtonCheckbox.tsx'
 ```
 
-### Split button (action, primary)
+### Split button toggle with text label
     
-```ts file='MenuToggleSplitButtonActionPrimary.tsx'
-```
+To display text in a menu toggle, add a label to the `items` property of `splitButtonOptions`.     
 
-### Split button (action, secondary)
-    
-```ts file='MenuToggleSplitButtonActionSecondary.tsx'
+```ts file='MenuToggleSplitButtonCheckboxWithText.tsx'
 ```
 
-### With icon/image and text
+### Split button toggle with variant styling 
 
-```ts
-import React from 'react';
-import { MenuToggle, Avatar } from '@patternfly/react-core';
-import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
+Variant styling can be applied to split button toggles to adjust their appearance for different scenarios. The following example applies “primary” styling to the toggles shown.
```

Comment:
It'd be worth mentioning that primary and secondary variants can be used with split button toggles.